app/services/expired_dossiers_deletion_service.rb

```diff
@@ -77,7 +77,7 @@ def self.delete_expired_en_construction_and_notify
   end
 
   def self.delete_expired_termine_and_notify
-    delete_expired_and_notify(Dossier.termine_expired)
+    delete_expired_and_notify(Dossier.termine_expired, true)
```

Comment:
Le `true` n'est pas très clair à la lecture, je trouve.

J'ai l'impression que c'est un cas où un named argument serait une bonne idée :

```suggestion
    delete_expired_and_notify(Dossier.termine_expired, notify_on_closed_procedure: true)
```